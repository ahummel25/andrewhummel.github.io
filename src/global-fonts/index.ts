import RobotoSansSerif from './KFOmCnqEu92Fr1Mu72xKKTU1Kvnz.woff2';

// injectGlobal`
// @font-face {
// 	font-family: 'Roboto-thin';
// 	font-style: normal;
// 	font-weight: 100;
// 	src: url('./KFOmCnqEu92Fr1Mu72xKKTU1Kvnz.woff2') format('woff2');
//   }
// `;

export { RobotoSansSerif };
