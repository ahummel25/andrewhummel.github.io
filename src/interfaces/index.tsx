import { MutableRefObject } from 'react';
import { IGatsbyImageData } from 'gatsby-plugin-image';

export interface IChildrenProp {
  children?: React.ReactNode;
}
export interface IImages {
  [key: string]: {
    childImageSharp: {
      gatsbyImageData: IGatsbyImageData;
    };
  };
}

export interface IWeatherResponse {
  coord: {
    lon: number;
    lat: number;
  };
  weather: {
    id: number;
    main: string;
    description: string;
    icon: string;
  }[];
  base: string;
  main: {
    temp: number;
    feels_like: number;
    temp_min: number;
    temp_max: number;
    pressure: number;
    humidity: number;
  };
  visibility: number;
  wind: {
    speed: number;
    deg: number;
    gust: number;
  };
  clouds: {
    all: number;
  };
  dt: number;
  sys: {
    type: number;
    id: number;
    country: string;
    sunrise: number;
    sunset: number;
  };
  timezone: number;
  id: number;
  name: string;
  cod: number;
}

export interface ForwardedRefsProps {
  refsToForward: {
    [key: string]: {
      ref: MutableRefObject<HTMLDivElement | null>;
      onClickHandler: () => void;
    };
  };
}

export interface LayoutMainProps extends IChildrenProp {
  paddingLeft: number;
}

export interface HeaderProps {
  // eslint-disable-next-line no-unused-vars
  setDrawerWidth: (drawerWidth: number) => void;
}

export interface NavbarProps {
  // eslint-disable-next-line no-unused-vars
  setDrawerWidth: (drawerWidth: number) => void;
}

export interface NavListProps {
  // eslint-disable-next-line no-unused-vars
  setOpen?: (open: boolean) => void;
}
