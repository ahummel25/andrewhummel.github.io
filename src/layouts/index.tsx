import React, { FC } from 'react';
import Helmet from 'react-helmet';
import { StaticQuery, graphql } from 'gatsby';
import styled from '@emotion/styled';

import 'materialize-css/dist/css/materialize.min.css';

interface StaticQueryProps {
  site: {
    siteMetadata: {
      author: {
        name: string;
      };
      faUrl: string;
      iconUrl: string;
      title: string;
      description: string;
      keywords: string;
      robotoFontsUrl: string;
    };
  };
}

const Main = styled.main`
  display: flex;
  flex-flow: column nowrap;
`;

const IndexLayout: FC<Record<string, unknown>> = ({ children }) => {
  return (
    <StaticQuery
      query={graphql`
        query IndexLayoutQuery {
          site {
            siteMetadata {
              faUrl
              iconUrl
              title
              description
              robotoFontsUrl
            }
          }
        }
      `}
      render={(data: StaticQueryProps): JSX.Element => (
        <Main>
          <Helmet
            htmlAttributes={{
              lang: 'en'
            }}
            title={data.site.siteMetadata.title}
            meta={[
              {
                name: 'description',
                content: data.site.siteMetadata.description
              },
              { name: 'keywords', content: data.site.siteMetadata.keywords }
            ]}
            link={[
              {
                href: data.site.siteMetadata.iconUrl,
                rel: 'stylesheet'
              },
              {
                href: data.site.siteMetadata.robotoFontsUrl,
                rel: 'stylesheet'
              },
              {
                href: data.site.siteMetadata.faUrl,
                rel: 'stylesheet'
              }
            ]}
          />
          {children}
        </Main>
      )}
    />
  );
};

export default IndexLayout;
