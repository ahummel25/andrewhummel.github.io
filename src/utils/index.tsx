import { RefObject } from 'react';

export const handleScrollTo = (elRef: RefObject<HTMLDivElement>): void => {
  if (elRef.current) {
    //Scroll the element into view
    elRef.current.scrollIntoView({
      behavior: 'smooth',
      block: 'start'
    });
  }
};
