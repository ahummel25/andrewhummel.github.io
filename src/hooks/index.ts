import console from 'console';

import { useEffect, useMemo, useState } from 'react';
import { graphql, useStaticQuery } from 'gatsby';

import { IImages, IWeatherResponse } from '../interfaces';
import { baseWeatherUrl } from '../services/api';
import { getFixedFahrenheit } from '../utils';

export const useGetFixedFahrenheit = (temp?: number): string | null => {
  const fixedTemp = useMemo(() => getFixedFahrenheit(temp), [temp]);
  return fixedTemp;
};

export const useGetImages = (): IImages => {
  const imgData = useStaticQuery(
    graphql`
      query {
        awsImg: file(relativePath: { eq: "aws.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        i11Img: file(relativePath: { eq: "i11.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        juristatImg: file(relativePath: { eq: "juristat.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        profileImg: file(relativePath: { eq: "me_full.jpg" }) {
          childImageSharp {
            fluid(quality: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        wwtImg: file(relativePath: { eq: "wwt.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        htmlImg: file(relativePath: { eq: "html5.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        reactImg: file(relativePath: { eq: "react.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        cssImg: file(relativePath: { eq: "css3.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        sassImg: file(relativePath: { eq: "sass.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 140, maxHeight: 105) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        bootstrapImg: file(relativePath: { eq: "bootstrap.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        materializeImg: file(relativePath: { eq: "materialize.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        styCompImg: file(relativePath: { eq: "styled_components.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        tsImg: file(relativePath: { eq: "typescript.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        jqueryImg: file(relativePath: { eq: "jquery.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        nodeImg: file(relativePath: { eq: "nodejs.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        pythonImg: file(relativePath: { eq: "python.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        goImg: file(relativePath: { eq: "golang.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        javaImg: file(relativePath: { eq: "java.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        mysqlImg: file(relativePath: { eq: "mysql.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        oracleImg: file(relativePath: { eq: "oracle.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
      }
    `
  );
  return imgData;
};

export const useGetWeatherByCoords = (
  units = 'imperial'
): IWeatherResponse | null => {
  let latitude = 0;
  let longitude = 0;
  const [
    weatherResponse,
    setWeatherResponse
  ] = useState<IWeatherResponse | null>(null);

  const positionOptions = { timeout: 60000, enableHighAccuracy: true };

  useEffect((): void => {
    const getWeatherByCoords = async (): Promise<void> => {
      const { coords }: Position = await new Promise((resolve): void => {
        if (navigator.geolocation) {
          navigator.geolocation.getCurrentPosition(
            (position: Position): void => {
              resolve(position);
            },
            (err: PositionError): void => {
              if (typeof window !== 'undefined') {
                console.debug(
                  `Error code ${err.code} while calling getCurrentPosition: ${err.message}`
                );
              }
            },
            positionOptions
          );
        } else {
          if (typeof window !== 'undefined') {
            console.debug('Sorry, your browser does not support geolocation!');
          }
        }
      });

      latitude = coords?.latitude;
      longitude = coords?.longitude;

      if (latitude && longitude) {
        const response = await fetch(
          `${baseWeatherUrl}/weather?lat=${latitude}&lon=${longitude}&appid=${process.env.GATSBY_WEATHER_API_KEY}&units=${units}`
        );
        const weather: IWeatherResponse = await response.json();

        setWeatherResponse(weather);
      }
    };

    getWeatherByCoords();
  }, []);

  return weatherResponse;
};

export const useWindowDimensions = (): { width: number; height: number } => {
  const getWindowDimensions = (): { width: number; height: number } => {
    let width = 0;
    let height = 0;
    if (typeof window !== 'undefined') {
      const { innerWidth, innerHeight } = window;
      width = innerWidth;
      height = innerHeight;
    }

    return {
      width,
      height
    };
  };
  const [windowDimensions, setWindowDimensions] = useState<{
    width: number;
    height: number;
  }>(getWindowDimensions());

  useEffect((): (() => void) => {
    const handleResize = (): void => {
      setWindowDimensions(getWindowDimensions());
    };

    window.addEventListener('resize', handleResize);
    return (): void => {
      window.removeEventListener('resize', handleResize);
    };
  }, []);

  return windowDimensions;
};
