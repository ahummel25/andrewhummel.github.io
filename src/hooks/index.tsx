import { useEffect, useMemo, useState } from 'react';
import { graphql, useStaticQuery } from 'gatsby';

import { IImages, IWeatherResponse } from '../interfaces';
import { baseWeatherUrl } from '../services/api';
import { getFixedFahrenheit } from '../utils';

export const useGetFixedFahrenheit = (temp?: number): string | null => {
  const fixedTemp = useMemo(() => getFixedFahrenheit(temp), [temp]);
  return fixedTemp;
};

export const useGetImages = (): IImages => {
  const imgData = useStaticQuery(
    graphql`
      query {
        awsImg: file(relativePath: { eq: "aws.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 100, maxHeight: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        i11Img: file(relativePath: { eq: "i11.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        juristatImg: file(relativePath: { eq: "juristat.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        profileImg: file(relativePath: { eq: "me_full.jpg" }) {
          childImageSharp {
            fluid(quality: 100) {
              ...GatsbyImageSharpFluid
            }
          }
        }
        wwtImg: file(relativePath: { eq: "wwt.png" }) {
          childImageSharp {
            fluid(quality: 100, maxWidth: 60, maxHeight: 60) {
              ...GatsbyImageSharpFluid
            }
          }
        }
      }
    `
  );
  return imgData;
};

export const useGetWeatherByCoords = (
  units = 'imperial'
): IWeatherResponse | null => {
  const [
    weatherResponse,
    setWeatherResponse
  ] = useState<IWeatherResponse | null>(null);

  useEffect(() => {
    let mounted = true;
    const getWeatherByCoords = async (): Promise<void> => {
      const {
        coords: { latitude, longitude }
      }: Position = await new Promise((resolve, reject): void => {
        if (navigator.geolocation) {
          navigator.geolocation.getCurrentPosition(
            (position: Position): void => {
              resolve(position);
            },
            (err: PositionError): void => {
              if (err.code === 1) {
                reject('Error: Access is denied!');
              } else if (err.code === 2) {
                reject('Error: Position is unavailable!');
              }
            },
            { timeout: 60000 }
          );
        } else {
          reject('Sorry, browser does not support geolocation!');
        }
      });

      if (mounted) {
        const response = await fetch(
          `${baseWeatherUrl}/weather?lat=${latitude}&lon=${longitude}&appid=${process.env.GATSBY_WEATHER_API_KEY}&units=${units}`
        );
        const weather: IWeatherResponse = await response.json();
        setWeatherResponse(weather);
      }
    };

    getWeatherByCoords();

    return (): void => {
      mounted = false;
    };
  }, []);

  return weatherResponse;
};

export const useWindowDimensions = (): { width: number; height: number } => {
  const getWindowDimensions = (): { width: number; height: number } => {
    let width = 0;
    let height = 0;
    if (typeof window !== 'undefined') {
      const { innerWidth, innerHeight } = window;
      width = innerWidth;
      height = innerHeight;
    }

    return {
      width,
      height
    };
  };
  const [windowDimensions, setWindowDimensions] = useState<{
    width: number;
    height: number;
  }>(getWindowDimensions());

  useEffect(() => {
    const handleResize = (): void => {
      setWindowDimensions(getWindowDimensions());
    };

    window.addEventListener('resize', handleResize);
    return (): void => {
      window.removeEventListener('resize', handleResize);
    };
  }, []);

  return windowDimensions;
};
