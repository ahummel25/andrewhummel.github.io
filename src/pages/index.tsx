import React, { useRef, useState } from "react";

import Page from "../components/Page";
import Profile from "../components/Profile";
import IndexLayout from "../layouts";
import Header from "../components/Header";
import LayoutMain from "../components/LayoutMain";
import { handleScrollTo } from "../utils";

/*interface StaticQueryProps {
  site: {
    siteMetadata: {
      author: {
        name: string;
      };
      iconUrl: string;
      title: string;
      description: string;
      keywords: string;
    };
  };
}*/

const IndexPage = (): JSX.Element => {
  const [drawerWidth, setDrawerWidth] = useState<number>(0);
  const aboutRef = useRef<HTMLDivElement | null>(null);
  const experienceRef = useRef<HTMLDivElement | null>(null);
  const projectsRef = useRef<HTMLDivElement | null>(null);
  const refsToForward = {
    about: {
      ref: aboutRef,
      onClickHandler: () => {
        handleScrollTo(aboutRef);
      }
    },
    experience: {
      ref: experienceRef,
      onClickHandler: () => {
        handleScrollTo(experienceRef);
      }
    },
    projects: {
      ref: projectsRef,
      onClickHandler: () => {
        handleScrollTo(projectsRef);
      }
    }
  };

  return (
    <IndexLayout>
      <LayoutMain paddingLeft={drawerWidth}>
        <Header refsToForward={refsToForward} setDrawerWidth={setDrawerWidth} />
        <Page>
          <Profile refsToForward={refsToForward} />
        </Page>
      </LayoutMain>
    </IndexLayout>
  );
};

export default IndexPage;
