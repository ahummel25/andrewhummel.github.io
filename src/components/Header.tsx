import React, { FC } from "react";
import styled from "@emotion/styled";
import Img from "gatsby-image";

import Navbar from "../components/Navbar";
import { useGetProfileImage } from "../hooks";
import { HeaderProps } from "../interfaces";

const ImgStyled = styled(({ ...rest }) => <Img {...rest} />)`
  @media (min-width: 992px) {
    margin-top: 0;
  }
  margin-top: 64px;
  display: inline-block;
`;

const Header: FC<HeaderProps> = ({
  refsToForward,
  setDrawerWidth
}): JSX.Element => {
  const imgData = useGetProfileImage();

  return (
    <>
      <Navbar refsToForward={refsToForward} setDrawerWidth={setDrawerWidth} />
      <ImgStyled
        fluid={imgData.file.childImageSharp.fluid}
        alt="Profile Shot"
      />
    </>
  );
};

export default Header;
