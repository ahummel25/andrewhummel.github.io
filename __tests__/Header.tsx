jest.mock('../src/hooks', () => ({
  useGetFixedFahrenheit: (temp: number): string => `${temp.toFixed()}`,
  useGetImages: (): IImages => ({
    profileImg: {
      childImageSharp: {
        fluid: {
          aspectRatio: 1.5664414414414414,
          base64:
            'data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj',
          src:
            '/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg',
          srcSet: `/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/my_mock_image.jpg 200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/my_mock_image.jpg 400w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg 800w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/my_mock_image.jpg 1200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/my_mock_image.jpg 1391w`,
          sizes: '(max-width: 800px) 100vw, 800px'
        }
      }
    }
  }),
  useWindowDimensions: (): { width: number; height: number } => ({
    height: 100,
    width: 100
  }),
  useGetWeatherByCoords: (): IWeatherResponse => ({
    coord: {
      lon: -87.66,
      lat: 41.89
    },
    weather: [
      {
        id: 804,
        main: 'Clouds',
        description: 'overcast clouds',
        icon: '04n'
      }
    ],
    base: 'stations',
    main: {
      temp: 52.48,
      feels_like: 48.56,
      temp_min: 48.99,
      temp_max: 55.99,
      pressure: 1004,
      humidity: 82
    },
    visibility: 16093,
    wind: {
      speed: 5.82,
      deg: 240,
      gust: 10
    },
    clouds: {
      all: 90
    },
    dt: 1584060065,
    sys: {
      type: 1,
      id: 4505,
      country: 'US',
      sunrise: 1584014806,
      sunset: 1584057231
    },
    timezone: -18000,
    id: 4887398,
    name: 'Chicago',
    cod: 200
  })
}));

import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../src/components/Header';
import { IImages, IWeatherResponse } from '../src/interfaces';

describe('Header', () => {
  const setDrawerWidth = jest.fn();

  it('renders correctly', () => {
    const tree = renderer
      .create(<Header setDrawerWidth={setDrawerWidth} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
