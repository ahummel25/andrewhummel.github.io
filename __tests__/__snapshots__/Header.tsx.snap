// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<section
  className="section scrollspy"
  id="header"
  style={
    Object {
      "display": "contents",
    }
  }
>
  <div
    className="MuiDrawer-root MuiDrawer-docked css-1uttx8n"
  >
    <div
      className="MuiPaper-root MuiDrawer-paper MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft MuiPaper-elevation0"
    >
      <div
        className="name-title"
      >
        <a
          href="/profile"
        >
          <h1>
            Andrew 
            <br />
             Hummel
          </h1>
          <span
            className="brown-text light"
          >
            Full Stack Developer
          </span>
        </a>
      </div>
      <hr
        className="MuiDivider-root"
      />
      <ul>
        <li
          className="css-19ff2z1"
          color="rgb(0, 128, 115)"
        >
          <a
            className="waves-effect waves-dark"
            href="#about"
            onClick={[Function]}
          >
            <i
              className="small material-icons icon-color"
            >
              person
            </i>
            About
          </a>
        </li>
        <li
          className="css-1gblma6"
          color="rgb(0, 188, 212)"
        >
          <a
            className="waves-effect waves-dark"
            href="#experience"
            onClick={[Function]}
          >
            <i
              className="small material-icons icon-color"
            >
              trending_up
            </i>
            Experience
          </a>
        </li>
        <li
          className="css-1lyki0f"
          color="rgb(156,38,176)"
        >
          <a
            className="waves-effect waves-dark"
            href="#skills"
            onClick={[Function]}
          >
            <i
              className="small material-icons icon-color"
            >
              equalizer
            </i>
            Skills
          </a>
        </li>
        <li
          className="css-1j8bumt"
          color="rgb(244,67,53)"
        >
          <a
            className="waves-effect waves-dark"
            href="#certifications"
            onClick={[Function]}
          >
            <i
              className="small material-icons icon-color"
            >
              star
            </i>
            Certifications
          </a>
        </li>
        <li
          className="css-djjvk3"
          color="rgb(121, 85, 72)"
        >
          <a
            className="waves-effect waves-dark"
            href="#contact"
            onClick={[Function]}
          >
            <i
              className="small material-icons icon-color"
            >
              mail
            </i>
            Contact
          </a>
        </li>
      </ul>
      <div
        className="css-1pvvq55"
      >
        <div
          className="css-31ceef"
        >
          <img
            alt="04n"
            className="css-ays64o"
            src="https://openweathermap.org/img/w/04n.png"
          />
          <div
            className="css-11b29mu"
          >
            Thu, Mar 12th
          </div>
        </div>
        <div
          className="css-jim3c1"
        >
          <strong>
            56
             ℉
          </strong>
          | 
          49
           ℉
        </div>
        <div>
          humidity: 
          82
           %
        </div>
        <div>
          speed: 
          5.82
           mps
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-19iflly gatsby-image-wrapper"
    style={
      Object {
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      aria-hidden={true}
      style={
        Object {
          "paddingBottom": "63.83896477354421%",
          "width": "100%",
        }
      }
    />
    <img
      alt=""
      aria-hidden={true}
      src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "objectFit": "cover",
          "objectPosition": "center",
          "opacity": 1,
          "position": "absolute",
          "top": 0,
          "transitionDelay": "500ms",
          "width": "100%",
        }
      }
    />
    <picture>
      <source
        sizes="(max-width: 800px) 100vw, 800px"
        srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/my_mock_image.jpg 200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/my_mock_image.jpg 400w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg 800w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/my_mock_image.jpg 1200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/my_mock_image.jpg 1391w"
      />
      <img
        alt="Profile Shot"
        loading="lazy"
        onLoad={[Function]}
        sizes="(max-width: 800px) 100vw, 800px"
        src="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg"
        srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/my_mock_image.jpg 200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/my_mock_image.jpg 400w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg 800w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/my_mock_image.jpg 1200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/my_mock_image.jpg 1391w"
        style={
          Object {
            "height": "100%",
            "left": 0,
            "objectFit": "cover",
            "objectPosition": "center",
            "opacity": 0,
            "position": "absolute",
            "top": 0,
            "transition": "opacity 500ms",
            "width": "100%",
          }
        }
      />
    </picture>
    <noscript
      dangerouslySetInnerHTML={
        Object {
          "__html": "<picture><source srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/my_mock_image.jpg 200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/my_mock_image.jpg 400w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg 800w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/my_mock_image.jpg 1200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/my_mock_image.jpg 1391w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/my_mock_image.jpg 200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/my_mock_image.jpg 400w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg 800w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/my_mock_image.jpg 1200w,
				  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/my_mock_image.jpg 1391w\\" src=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/my_mock_image.jpg\\" alt=\\"Profile Shot\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
        }
      }
    />
  </div>
</section>
`;
